src/main/java/uk/gov/ch/developer/docs/controller/developer/SignOutController.java

```diff
@@ -32,8 +30,9 @@ public void doSignOut(final HttpServletResponse httpServletResponse,
 
         final Session chSession = (Session) httpServletRequest
                 .getAttribute(SessionHandler.CHS_SESSION_REQUEST_ATT_KEY);
-        oauth2.invalidateSession(chSession);
-        httpServletResponse.sendRedirect(identityProviders.getRedirectUriPage());
+        String redirectUri = ioAuthCoordinator.getSignoutUri();
+        ioAuthCoordinator.invalidateSession(chSession);
+        httpServletResponse.sendRedirect(redirectUri);
```

Comment:
I'd rather that we just use the "redirect:"+ redirectUri
It removes the IOException from this class and allows Spring to handle it.

Reply:
Do you mean have the controller method return a String with that in it?